manim/mobject/graphing/probability.py

```diff
@@ -468,11 +504,17 @@ def construct(self):
                     # if already negative, then we move the bottom edge of the bar
                     # to the location of the previous top
 
-                bar.stretch_to_fit_height(quotient * bar.height)
+                bar.stretch_to_fit_height(abs(quotient) * bar.height)
 
             except ZeroDivisionError:
```

Comment:
Can we change this to not use try/except. I mean we know when theres a division by zero error. No need to pull up a whole try/except. if else should do the job 🤷🏼

Reply:
Sure, done.